.github/workflows/main.yml

```diff
@@ -21,10 +21,10 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: '14.5.0'
+          node-version: '14.16.0'
 
       - name: Install
-        run: npm install
+        run: npm ci
```

Comment:
I discovered this while trying to debug the CI failures. This is supposed to be faster and is recommended by GitHub.  It uses only the versions specified in package-lock.json.